client/components/vertical-menu/README.md

```diff
@@ -0,0 +1,40 @@
+# Vertical Menu
+
+This component provides a vertically-oriented list of items for typical use in selecting between different views in another part of the visible screen.
+
+It is designed to take a list of React components using the `vertical-menu__items` CSS class. Of course, any special and standard type of list item should be implemented until `items/` as its own component.
```

Comment:
typo: "should be implemented until `items/` as its own component." The meaning isn't clear to me.